packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -131,9 +135,27 @@ const FinancialDetails = (props: TFinancialDetails & TFinancialInformationAndTra
                                     height_offset='110px'
                                     is_disabled={isDesktop()}
                                 >
-                                    <Text as='p' color='prominent' size='xxs' className='trading-assessment__side-note'>
-                                        <Localize i18n_default_text='We collect information about your employment as part of our due diligence obligations, as required by anti-money laundering legislation.' />
-                                    </Text>
+                                    {is_eu && (
```

Comment:
is_eu is kind of deprecate. Can you try to check is_eu_user from traders_hub store?

Reply:
sure let me check on it 👌